src/seedu/addressbook/data/person/Phone.java

```diff
@@ -8,9 +8,9 @@
  */
 public class Phone {
 
-    public static final String EXAMPLE = "123456789";
-    public static final String MESSAGE_PHONE_CONSTRAINTS = "Person phone numbers should only contain numbers";
-    public static final String PHONE_VALIDATION_REGEX = "\\d+";
+    public static final String EXAMPLE = "123456789 or 1234 5678";
+    public static final String MESSAGE_PHONE_CONSTRAINTS = "Person phone numbers should only contain numbers and whitespace(s)";
```

Comment:
In addition to updating the message displayed to the user, you should also update the User Guide.